ovc/resource_ovc_machine.go

```diff
@@ -349,11 +374,58 @@ func resourceOvcMachineUpdate(d *schema.ResourceData, m interface{}) error {
 		} else {
 			// delete all interfaces
 			log.Println("[DEBUG] Detaching from all external networks")
-			if err := client.Machines.DeleteExternalIP(machineIDInt, 0, ""); err != nil {
+			var emptyIP string
+			if err := client.Machines.DeleteExternalIP(machineIDInt, 0, emptyIP); err != nil {
```

Comment:
Why not leave it at `client.Machines.DeleteExternalIP(machineIDInt, 0, "")` as `emptyIP` is never set with a new value?